README.md

```diff
@@ -1,26 +1,78 @@
+<!-- markdownlint-disable MD026 MD033 -->
+
 # DiscordFeedback
-The bot that powers the Discord Feedback server
 
-# Selfhosting
-We don't (fully) support hosting your own copy of DiscordFeedback, this program is highly customized and not suited to run outside of the official Discord Feedback server.
+This is the bot that powers the Discord Feedback server, develop'd by the volunteer group only known as "Discord Feedback Bot Tinkerers Sqaud™"
+
+## Table of Contents
+
+- [Development](#development)
+  - [Config](#config)
+  - [Global ESlint Install](#global-eslint-instal)
+  - [Testing](#coming-soon)
+- [Selfhosting](#selfhosting)
+  - [Config Creation](#config-creation)
+  - [Database Initalization](#database-initialization)
+  - [Running the Bot](#running-the-bot)
+
```

Comment:
This isn't like the Magna Carta, do we really need an index if the file is less than 100 lines?

Reply:
I mean, its more for quick reference, but I can remove it if that's desired.